### PR TITLE
Dictionary consistency with DPR v1

### PR DESCRIPTION
**Pull request by parthopdas**
Please see commit by commit. You can ignore the first commit as that just unformats the documents.

### CODE REVIEW

File: digitalpalireader/content/js/dict_xml.js [hunk 1]
```diff
@@ -1,796 +1,634 @@
...
+		dataNode.innerHTML = data;
+		document.getElementById('difb').setAttribute('align','left');
+		document.getElementById('difb').appendChild(dataNode);
+		document.getElementById('cdif').scrollTop=0;
+	} else {
+    const difbId = `#${DPR_PAL.getDifId()}`;
+		var dataNode = $('<div></div>').html(data);
+		$(difbId).html('');
+		$(difbId).append(dataNode);
+		$('#cdif').scrollTop(0);
```
Comment (dangeol) [inline digitalpalireader/content/js/dict_xml.js]:
```suggestion
		$('#paliTextContent').scrollTop(0);
```

File: digitalpalireader/content/js/dict_xml.js [hunk 2]
```diff
@@ -1,796 +1,634 @@
...
+	xmlhttp.send(null);
+	var xmlDoc = xmlhttp.responseXML.documentElement;
+	var s = new XMLSerializer();
+	var data = s.serializeToString(xmlDoc);
+
+	data = data.replace(/<p><a href="index.htm">Index<\/a><\/p>/,'').replace(/<a href="([^.]+)[^>]+/g,"<a class=\"green\" href=\"javascript:void(0)\" onclick=\"sktRXML('$1')\"");
+
+	// output
+
+    displayDictData(data);
```
Comment (dangeol) [inline digitalpalireader/content/js/dict_xml.js]: Clicking on a sanskrit root result item does not work locally in Chrome and Firefox. For loading `html` files, for me only an asynchronous request such below works, but there might be easier workarounds than this.

```  
  var data;
  var file = DPR_PAL.baseUrl + 'sanskrit/content/skt-roots/'+sktR[no]+'.htm';
  var xmlhttp = new window.XMLHttpRequest();
  var xmlhttp = new XMLHttpRequest();
  xmlhttp.onload = function () {
    var s = new XMLSerializer();
    var xmlDoc = xmlhttp.responseXML.documentElement;
    data = s.serializeToString(xmlDoc);
    data = data.replace(/<p><a href="index.htm">Index<\/a><\/p>/,'').replace(/<a href="([^.]+)[^>]+/g,"<a class=\"green\" href=\"javascript:void(0)\" onclick=\"sktRXML('$1')\"");
    displayDictData(data);
  }
  xmlhttp.onerror = function () {
    dump('Error while getting XML.');
  }
  xmlhttp.open('GET', file);
  xmlhttp.responseType = 'document';
  xmlhttp.send()0;
```

Comment (parthopdas) [reply digitalpalireader/content/js/dict_xml.js]: yeah html cannot be loaded by sync requests - this is as per the API documentation. 
i renamed the file to xml and it was working fine.

that is the fix i am working on currently - ie renaming all files to xml

File: digitalpalireader/content/js/dict_xml.js (hunk 1 above)
Comment (parthopdas) [reply digitalpalireader/content/js/dict_xml.js]: thanks for catching this. i'll make this dynamic as well.

right after alpha - i want to discuss with you my ideas for modernizing the code base. one of which is to do away with the hardcoded html IDs.

**Pull request merged**